schema/Retail/RET/on_search.ts

```diff
@@ -88,7 +88,7 @@ export const onSearchSchema = {
                   },
                   type: {
                     type: 'string',
-                    enum: ['Delivery', 'Self-Pickup', 'Delivery and Self-Pickup'],
+                    enum: ['Delivery', 'Self-Pickup', 'Buyer-Delivery'],
```

Comment:
bpp/fulfillments is the legacy list and it includes Delivery and Self-Pickup option. Buyer Delivery should be provided in bpp/providers[]/fulfillments

Reply:
add the enums for bpp/providers[]/fulfillments